frontend/packages/pipelines-plugin/src/components/repository/repository-form-utils.ts

```diff
@@ -123,10 +129,11 @@ export const createRepositoryResources = async (
     },
   };
 
-  const resource = await k8sCreateResource({
+  const resource = k8sCreateResource({
```

Comment:
Huh? Why we don't want to wait here anymore???

Reply:
Sure. Reverted